README.md

```diff
@@ -1,7 +1,7 @@
 # About
 This SDK is meant to help users with basic queries to WattTime’s API (version 3), and to get data returned in specific formats (e.g., JSON, pandas, csv).
 
-Users must first [register for access to the WattTime API here](https://watttime.org/docs-dev/data-plans/).
+Users may register for access to the WattTime API through this client, however the basic user scoping given will only allow newly registered users to access data for the `CAISO_NORTH` region. Additionally, data may not be available for all signal types for newly registered users.
```

Comment:
Do we still have an index datatype? or is CAISO_NORTH the only free thing in the new API?

Reply:
Still tbd afaik. Index is not implemented on v3 yet, and unclear if it will be.